Volume/Views/Flyers/FlyerCellPast.swift

```diff
@@ -0,0 +1,201 @@
+        HStack(alignment: .top) {
+            Text(flyer.organization.name)
+                .font(Constants.organizationNameFont)
+                .lineLimit(2)
+            
+            Spacer()
+            
+            bookmarkButton
+            shareButton
+        }
+        .padding(.bottom, -Constants.verticalSpacing)
+    }
+    
+    private var flyerTitle: some View {
+        Text(flyer.title)
+            .font(Constants.titleFont)
+            .lineLimit(1)
+    }
+    
+    private var flyerDate: some View {
+        HStack {
+            Image.volume.calendar
+                .foregroundColor(Color.black)
+            
+            Text(flyer.date.start.flyerDateString)
+                .font(Constants.dateFont)
+                .padding(.trailing, Constants.horizontalSpacing)
+                .lineLimit(1)
+            
+            Text("\(flyer.date.start.flyerTimeString) - \(flyer.date.end.flyerTimeString)")
+                .font(Constants.dateFont)
+                .lineLimit(1)
+        }
+    }
+    
+    private var flyerLocation: some View {
+        HStack {
+            Image.volume.location
+                .foregroundColor(Color.black)
+            
+            Text(flyer.location)
+                .font(Constants.locationFont)
+                .lineLimit(1)
+        }
+    }
+    
+    private var categoryType: some View {
+        Text(Organization.contentTypeString(
+                type: flyer.organization.contentType)
+            )
+            .padding(.init(
+                top: Constants.categoryVerticalPadding,
+                leading: Constants.categoryHorizontalPadding,
+                bottom: Constants.categoryVerticalPadding,
+                trailing: Constants.categoryHorizontalPadding
+            ))
+            .font(Constants.categoryFont)
+            .foregroundColor(Color.volume.orange)
+            .background(Color.white)
+            .overlay(
+                RoundedRectangle(cornerRadius: Constants.categoryCornerRadius)
+                    .stroke(Color.volume.orange, lineWidth: 1)
+            )
+    }
+    
+}
+
+extension FlyerCellPast {
+    
+    struct Skeleton: View {
+        var body: some View {
+            HStack {
+                SkeletonView()
+                    .frame(width: Constants.imageWidth, height: Constants.imageHeight)
+                
+                VStack(alignment: .leading) {
+                    SkeletonView()
+                        .frame(width: 130, height: 15)
+                    
+                    SkeletonView()
+                        .frame(width: 200, height: 20)
+                    
+                    SkeletonView()
+                        .frame(width: 180, height: 15)
+                    
+                    SkeletonView()
+                        .frame(width: 100, height: 15)
+                    
+                    SkeletonView()
+                        .frame(width: 60, height: 15)
```

Comment:
The reason why I have different frame dimensions is because each `SkeletonView` represents a different gray bar (for example, I can't know for sure how long a text is). For the ones that do end up showing the image, I will double check the dimension and make sure it lines up nicely.